host/tests/helpers/dsAudioTestHelper.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+import yaml
+import os
+import sys
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(dir_path+"/../")
+
+from raft.framework.plugins.ut_raft.utHelper import utHelperClass
+from raft.framework.plugins.ut_raft.ut_core import UTCoreMenuNavigator
+from raft.framework.core.outboundClient import outboundClientClass
+
+#TODO: Read from yaml file
+run_script="./bins/run.sh"
+
+class dsAudioTestHelperClass(utHelperClass):
+
+    UT = None
+    test_config = None
+    menu_yaml = None
+
+    """
+    Device Settings Audio Test Helper Code
+
+    This module provides common extensions for device Settings Audio Module.
+    """
+    def __init__(self, testName, qcId, test_config_path, log=None ):
+        super().__init__(testName, qcId, log=log )
+
+        test_config = self.load_yaml(test_config_path)
+        working_directory = test_config["dsAudio"]["test_prerequisite"]["Working_directory"]
+
+        outBoundClient = outboundClientClass(log, working_directory)
+
+        # Copy test binaries to the target
+        for source,destination in zip(test_config["dsAudio"]["test_prerequisite"]["test_bin"]["source"], test_config["dsAudio"]["test_prerequisite"]["test_bin"]["destination"]):
+            file_name = os.path.basename(source)
+            outBoundClient.downloadFile(source, file_name)
+            self.copyFileFromHost(os.path.join(working_directory, file_name), destination)
```

Comment:
need a global method in ut-helper to support copying files to the box, it should also download them, therefore removing the problem from here..

opkg format should be used really also.